docs/usage.rst

```diff
@@ -3,19 +3,60 @@ Usage
 =====
 
 
-`ansible-rulebook` is either used as a command line tool or used as a web service with `ansible-rulebook-ui`.  
+`ansible-rulebook` is either used as a command line tool or used as a web service with `ansible-rulebook-ui`.
+
+
+======================
+CLI Usage and Examples
+======================
+
+
+The `ansible-rulebook` CLI supports the following options::
+
+    Options:
+    -h, --help                  Show this page
+    -i, --inventory=<i>         Inventory
+    --rulebook=<r>              The rulebook file or rulebook from a collection
+    -S=<S>, --source_dir=<S>    Source dir
+    --vars=<v>                  Variables file
+    --env-vars=<e>              Comma separated list of variables to import
+                                from the environment
+    --debug                     Show debug logging
+    --verbose                   Show verbose logging
+    --version                   Show the version and exit
+    --websocket-address=<w>     Connect the event log to a websocket
+    --id=<i>                    Identifier
+    --worker                    Enable worker mode
+    --project-tarball=<p>       Project tarball
 
 To get help from `ansible-rulebook` run the following::
 
     ansible-rulebook --help
 
+To check the version of `ansible-rulebook` run the following::
+
+    ansible-rulebook --version
+
 The normal method for running `ansible-rulebook` is the following::
 
-    ansible-rulebook --inventory inventory.yml --rulebook rules.yml
+    ansible-rulebook --inventory inventory.yml --rulebook rules.yml --vars vars.yml
 
-If you are using custom event source plugins use the following::
+.. note::
+    The `--rulebook` option requires the `--inventory` option. The `--vars` option is not required.
 
+If you are using custom event source plugins use the following::
 
     ansible-rulebook --inventory inventory.yml --rulebook rules.yml -S sources/
 
-Here `sources` is a directory containing your event source plugins.
+.. note::
+    Here `sources` is a directory containing your event source plugins.
+
+If `eda-server` is running `ansible-rulebook` the `--worker`, `--id`, and `--websocket-address` options should be used::
```

Comment:
I would remove any mention to EDA-server, as it can be used independently. I would just describe that does the flags.

Reply:
I can do that but I thought these flags were only used when also using `eda-server`. Are there use cases where these specific flags are used without `eda-server`?

Reply:
Well, websocket is a way to expose the event stream data, can be used in multiple ways.  

Anyway, even if those flags are intended for eda-server as this project is independent, I only would explain what they do. What do you think @benthomasson?

Reply:
@Alex-Izquierdo thank you for the explanation! I removed the references to `eda-server` - take another look and make sure what I did makes sense!